libraries/3rdparty/mbedtls_utils/mbedtls_error.c

```diff
@@ -220,1364 +220,1117 @@ const char * mbedtls_strerror_highlevel( int errnum )
+
+            #if defined( MBEDTLS_PKCS12_C )
+                case -( MBEDTLS_ERR_PKCS12_BAD_INPUT_DATA )
+                    rc = "PKCS12 - Bad input parameters to function";
+                    break;
+
+                case -( MBEDTLS_ERR_PKCS12_FEATURE_UNAVAILABLE )
+                    rc = "PKCS12 - Feature not available, e.g. unsupported encryption scheme";
+                    break;
+
+                case -( MBEDTLS_ERR_PKCS12_PBE_INVALID_FORMAT )
+                    rc = "PKCS12 - PBE ASN.1 data not as expected";
+                    break;
+
+                case -( MBEDTLS_ERR_PKCS12_PASSWORD_MISMATCH )
+                    rc = "PKCS12 - Given private key password does not allow for correct decryption";
+                    break;
+            #endif /* MBEDTLS_PKCS12_C */
+
+            #if defined( MBEDTLS_PKCS5_C )
+                case -( MBEDTLS_ERR_PKCS5_BAD_INPUT_DATA )
+                    rc = "PKCS5 - Bad input parameters to function";
+                    break;
+
+                case -( MBEDTLS_ERR_PKCS5_INVALID_FORMAT )
+                    rc = "PKCS5 - Unexpected ASN.1 data";
+                    break;
+
+                case -( MBEDTLS_ERR_PKCS5_FEATURE_UNAVAILABLE )
+                    rc = "PKCS5 - Requested encryption or digest alg not available";
+                    break;
+
+                case -( MBEDTLS_ERR_PKCS5_PASSWORD_MISMATCH )
+                    rc = "PKCS5 - Given private key password does not allow for correct decryption";
+                    break;
+            #endif /* MBEDTLS_PKCS5_C */
+
+            #if defined( MBEDTLS_RSA_C )
+                case -( MBEDTLS_ERR_RSA_BAD_INPUT_DATA )
+                    rc = "RSA - Bad input parameters to function";
+                    break;
+
+                case -( MBEDTLS_ERR_RSA_INVALID_PADDING )
+                    rc = "RSA - Input data contains invalid padding and is rejected";
+                    break;
+
+                case -( MBEDTLS_ERR_RSA_KEY_GEN_FAILED )
+                    rc = "RSA - Something failed during generation of a key";
+                    break;
+
+                case -( MBEDTLS_ERR_RSA_KEY_CHECK_FAILED )
+                    rc = "RSA - Key failed to pass the validity check of the library";
+                    break;
+
+                case -( MBEDTLS_ERR_RSA_PUBLIC_FAILED )
+                    rc = "RSA - The public key operation failed";
+                    break;
+
+                case -( MBEDTLS_ERR_RSA_PRIVATE_FAILED )
+                    rc = "RSA - The private key operation failed";
+                    break;
+
+                case -( MBEDTLS_ERR_RSA_VERIFY_FAILED )
+                    rc = "RSA - The PKCS#1 verification failed";
+                    break;
+
+                case -( MBEDTLS_ERR_RSA_OUTPUT_TOO_LARGE )
+                    rc = "RSA - The output buffer for decryption is not large enough";
+                    break;
+
+                case -( MBEDTLS_ERR_RSA_RNG_FAILED )
```

Comment:
this doesn't look syntactically correct, there should be a colon after the case statement

Reply:
Fixed